### ISSUE TITLE
Old images for bitwarden/setup are not removed by script

### ISSUE THREAD

**Issue by asesidaa** [bitwarden/server#1758]
### Steps To Reproduce

1. Update bitwarden using `bitwarden.sh`
2. After upgrade, old images are removed, but the old bitwarden/setup images are not removed.

### Expected Result

Old image of bitwarden/setup is also removed.

### Actual Result

Old image of bitwarden/setup is not removed

### Additional Context

It seems that when removing old images, the bitwarden/setup images are filtered out, I wonder if the old images can be safely removed after upgrade.

### Build Version

1.44.1

### Environment

Self-Hosted

**Comment by Hinton** [comment]
@joseph-flinn Do you have any thoughts as to why we don't remove the `setup` image? At a quick glance it seems the other containers are removed in the restart command, but since setup isn't included in the `down` it gets skipped.

https://github.com/bitwarden/server/blob/master/scripts/run.ps1#L81-L83

**Comment by kspearrin** [comment]
Yes. We do this on purpose. Since `setup` is not an actively running container at the time this script is executing, I believe it was getting removed. Perhaps there is some other docker command we can use to be more smart about pruning older setup versions.

**Comment by closebot-bw** [comment]
⚠️ **Stale Issue Notice**

This issue has been automatically marked as stale due to inactivity. It will be closed in **2 weeks** (October 15, 2025) if no further activity occurs.

If this issue is still relevant and you would like to keep it open, please:
- Comment on this issue to show continued interest
- Provide any additional information or updates
- Confirm that the issue still exists in the latest version

Thank you for your contribution to this project! 🙏

**Comment by Mindless999** [comment]
I can confirm that the issue still exists in the latest version. I just updated and dit a docker images, where I can see images which are 2 years old (about the time when I first installed it)
